modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -6,16 +6,16 @@
 [id="rosa-hcp-classic-comparison_{context}"]
 = Comparing ROSA Classic and hosted control planes for ROSA
 
-{hcp-first} for {product-title} (ROSA) offers a different way to create a managed ROSA cluster. {hcp-first} for ROSA offers a reduced-cost solution with focuses on reliability and efficiency. With a focus on efficiency, you can quickly create a new cluster and deploy applications in minutes. 
+{hcp-title-first} offers a different way to create a managed ROSA cluster. {hcp-title} offers a reduced-cost solution with focuses on reliability and efficiency. With a focus on efficiency, you can quickly create a new cluster and deploy applications in minutes. 
 
-{hcp-first} for ROSA requires only a minimum two nodes, making it ideal for smaller projects, while still being able to scale to support larger projects and enterprises.
+{hcp-title} requires only a minimum two nodes, making it ideal for smaller projects, while still being able to scale to support larger projects and enterprises.
 
 .ROSA architectures comparison table
 
 [cols="3a,8a,8a",options="header"]
 |===
 | {nbsp} +
-| Hosted Control Plane 
+| Hosted Control Planes 
 | Classic
 
 | *What are each of the installation paths?*
```

Comment:
Down at line 38, it should say "a minimum of" not just "minimum of" to match line 37.

I'm also not sure about the numbers rules for our style. Does this rule mean that the numbers in the table should be spelled out or is it different because it is in a table?
<img width="811" alt="Screenshot 2023-04-28 at 12 04 01 PM" src="https://user-images.githubusercontent.com/119696027/235209620-d53126a8-4dc7-48af-9813-2715b4560508.png">